### DISCUSSION TITLE
**TALText** incorrectly to display font.family **entypo**

### DISCUSSION THREAD

**Discussion by pejuangdelphi** [MagicFoundation/Alcinoe#190] [Q&A]
@Zeus64 

**TALText** incorrectly to display font.family **entypo**

[Entypo.zip](https://github.com/Zeus64/alcinoe/files/6509724/Entypo.zip)

**Comment by Zeus64** [comment]
In with OS it's incorrectly display font.family entypo, on windows, ios or android ?

**Reply by pejuangdelphi** [reply]
On **Android**

**Reply by Zeus64** [reply]
I don't think the probleme is in TALText as on android we use the OS API. Can you try to see if you can correctly see the entypo font in a TEdit for example ?

**Reply by pejuangdelphi** [reply]
I have tried to implement in two components.
#1 TLabel
#2 TALText

TLabel shows correctly and TALText shows incorrectly.

TALText shows correctly at design-time only

**Reply by Zeus64** [reply]
Did you create a folder named font inside the res folder and copy your font

![image](https://user-images.githubusercontent.com/4325047/118879862-8270ed00-b8f1-11eb-8a7d-a170f6dc1780.png)

All font names must be only: lowercase a-z, 0-9, or underscore.
